src/main/java/io/stargate/sgv2/jsonapi/service/operation/filters/table/codecs/JSONCodecRegistry.java

```diff
@@ -67,22 +68,29 @@ public static <JavaT, CqlT> JSONCodec<JavaT, CqlT> codecToCQL(
   }
 
   public static <JavaT, CqlT> JSONCodec<JavaT, CqlT> codecToJSON(
-      TableMetadata table, CqlIdentifier column)
+      TableMetadata table, CqlIdentifier columnId)
       throws UnknownColumnException, MissingJSONCodecException {
 
     Preconditions.checkNotNull(table, "table must not be null");
-    Preconditions.checkNotNull(column, "column must not be null");
+    Preconditions.checkNotNull(columnId, "column must not be null");
```

Comment:
Shouldn't these be handled in the resolvers? These throws unhandled errors.

Reply:
Depends on how we view this: if there is no way to trigger it from outside (which I think is the case), these are assertions for internal state and should be ok this way.

But if there was a way to get null here from request it should throw `JsonApiException`.